stubs/pyinstaller/PyInstaller/utils/hooks/conda.pyi

```diff
@@ -0,0 +1,53 @@
+# https://pyinstaller.org/en/stable/hooks.html?highlight=conda_support#module-PyInstaller.utils.hooks.conda
+
+import sys
+from _typeshed import StrOrBytesPath
+from collections.abc import Sequence
+from pathlib import Path, PurePath
+from typing import Any
+from typing_extensions import TypeAlias, TypedDict
+
+if sys.version_info >= (3, 8):
+    from importlib.metadata import PackagePath as _PackagePath
+else:
+    _PackagePath: TypeAlias = Any
+
+CONDA_ROOT: Path
+CONDA_META_DIR: Path
+PYTHONPATH_PREFIXES: list[Path]
+
+class _RawDict(TypedDict):
+    name: str
+    version: str
+    files: list[StrOrBytesPath | PurePath]
+    depends: list[str]
+
+class Distribution:
+    raw: _RawDict
+    name: str
+    version: str
+    files: list[PackagePath]
+    dependencies: list[str]
+    packages: list[str | None]
```

Comment:
Not sure the items in `self.packages` can be `None`: `self.packages` is set [here](https://github.com/pyinstaller/pyinstaller/blob/36aa18acf2a04a4f28e45f9e805480f0df0c577a/PyInstaller/utils/hooks/conda.py#L103), and there's a check to make sure that `None` items are not appended to the list [here](https://github.com/pyinstaller/pyinstaller/blob/36aa18acf2a04a4f28e45f9e805480f0df0c577a/PyInstaller/utils/hooks/conda.py#L137-L138).

```suggestion
    packages: list[str]
```